homeassistant/components/transmission/__init__.py

```diff
@@ -115,6 +115,8 @@ def __init__(self, hass, config, api):
         self.started_torrents = []
         self.hass = hass
 
+        self.update()
```

Comment:
We try to avoid side effects in init method. Please move this to `setup` function.

Reply:
I think this is also not the correct fix. We should instead use our dispatch helper to signal the platform entities when we've updated the api in the component.

See comment here too:
https://github.com/home-assistant/home-assistant/pull/20575#issuecomment-460331798